src/modules/cmdpal/exts/Microsoft.CmdPal.Ext.Shell/FallbackExecuteItem.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CmdPal.Ext.Shell.Commands;
+using Microsoft.CmdPal.Ext.Shell.Helpers;
+using Microsoft.CmdPal.Ext.Shell.Pages;
+using Microsoft.CmdPal.Ext.Shell.Properties;
+using Microsoft.CmdPal.Extensions;
+using Microsoft.CmdPal.Extensions.Helpers;
+
+namespace Microsoft.CmdPal.Ext.Shell;
+
+internal sealed partial class FallbackExecuteItem : FallbackCommandItem
+{
+    private readonly ExecuteItem _executeItem;
+
+    public FallbackExecuteItem(SettingsManager settings)
+        : base(new ExecuteItem(string.Empty, settings))
+    {
+        _executeItem = (ExecuteItem)this.Command!;
+        Title = string.Empty;
```

Comment:
Something I think I've noticed in various places in the past -- I'm not sure if this is true still -- is that in general list items with no title are ghosts in the UI -- you can't see them but when you do up and down arrows the focus on the list item may just disappear. I'll test this out here too, may not happen but making a note so I check it out in future